docs/changes/0.19.0.rst

```diff
@@ -0,0 +1,74 @@
+Changelog for QCoDeS 0.19.0
+===========================
+
+The October 2020 release of QCoDeS.
+
+
+Breaking Changes:
+_________________
+
+There are no breaking changes in this release of QCoDeS.
+
+
+New:
+____
+-  Notebook: Example of how buffered sweep could be done on param with setpoints (#2263)
+
+
+Improved:
+_________
+- Station: Take verbose error message when validating station yaml against schema (#2282)
+- Legacy: ensure that datasets have unique names to avoid potential race condition (#2276) 
```

Comment:
This is really only a test thing so I suggest moving this to under the hood

Reply:
just want to confirm you're talking about PR #2276 right?

Reply:
yes